app/api/v2/views.py

```diff
@@ -59,4 +59,173 @@ def get(self):
+    
+    def post(self):
+        PARSER.parse_args()
+        data = request.get_json(silent=True)
+        posted = (data['id'], data['type'], data['location'],
+                  data['Images'], data['Videos'], data['comment'])
+        db.create_intervention_record(posted)
+        return{"status": 201, "data": [{"id": data['id'],
+                                        "message":"Created intervention record"}]}, 201
+
+
+class Intervention(Resource):
+    def get(self, intervention_id):
+        intervention = db.get_specific_intervention_record(intervention_id)
+        if intervention:
+            return make_response(jsonify({
+                "status": 200,
+                "data": intervention
+            }), 200)
+        return {"status": 404,
+                "data": [{
+                    "message": "intervention record does not exist."
+                }]}, 404
+    
+    def delete(self, intervention_id):
+        intervention = db.findOne(intervention_id)
+        if intervention:
+            db.delete_specific_record(intervention_id)
+            return make_response(jsonify({
+                "status": 200,
+                "data": [{"id": intervention_id,
+                          "message": "Intervention record has been deleted"}]
+            }), 200)
+        return {"status": 404,
+                "data": [{
+                    "message": "intervention record does not exist."
+                }]}, 404
+
+
+class Updatecomment(Resource):
+    def patch(self, intervention_id):
+        paserrr = reqparse.RequestParser(bundle_errors=True)
+        paserrr.add_argument('comment',
+                             type=valid_characters,
+                             required=True,
+                             help="comment field cannt be left blank or"
+                             "{error_msg},400"
+                             )
+        paserrr.parse_args()
+        data = request.get_json(silent=True)
+        comment = data["comment"]
+        patched = (comment, intervention_id)
+        intervention = db.findOne(intervention_id)
+        if intervention:
+            db.update_intervention_comment(patched)
+            return{"status": 200, "data":
+                   [{"id": intervention_id, "message":
+                     "Updated intervention's comment"}]}, 200
+        return {"status": 404,
+                "data": [{
+                    "message": "intervention record does not exist."
+                }]}, 404
+
+
+class Updatelocation(Resource):
+    def patch(self, intervention_id):
+        paserr = reqparse.RequestParser(bundle_errors=True)
+        paserr.add_argument('location',
+                            type=valid_characters,
+                            required=True,
+                            help="location field cannt be left blank or"
+                            "{error_msg},400"
+                            )
+        paserr.parse_args()
+        data = request.get_json(silent=True)
```

Comment:
change the variable name to something meaningful and make use of parser when getting data